### PR TITLE
adding bytes received/sent to cluster TCP stats

### PR DESCRIPTION
**Pull request by shovalyechiel1**
Commit Message: adding bytes received/sent to cluster TCP stats
Additional Description: Adding more tcp stats taken from tcpi_bytes_received and tcpi_bytes_sent data
Risk Level: Low
Testing: Compiled and tested with tcpdump for comparison
Docs Changes: No docs changed.
Release Notes: None
Platform Specific Features: None

### CODE REVIEW

**Review by ggreenway — CHANGES_REQUESTED** [review]
This seems reasonable, but you'll need to fix DCO: https://github.com/envoyproxy/envoy/blob/main/CONTRIBUTING.md#fixing-dco

/wait

File: docs/root/_include/tcp_stats.rst [hunk 1]
```diff
@@ -12,6 +12,8 @@
    cx_tx_data_segments, Counter, Total TCP segments with a non-zero data length transmitted
    cx_rx_data_segments, Counter, Total TCP segments with a non-zero data length received
    cx_tx_retransmitted_segments, Counter, Total TCP segments retransmitted
+   cx_rx_bytes_received, Counter, Bytes received from clients which TCP acknowledgments have been generated.
```
Comment (ggreenway) [inline docs/root/_include/tcp_stats.rst:15]:
```suggestion
   cx_rx_bytes_received, Counter, Total payload bytes received for which TCP acknowledgments have been sent.
```

File: docs/root/_include/tcp_stats.rst [hunk 2]
```diff
@@ -12,6 +12,8 @@
...
+   cx_tx_bytes_sent, Counter, Bytes transmitted from clients which TCP acknowledgments have been generated.
```
Comment (ggreenway) [inline docs/root/_include/tcp_stats.rst:16]:
```suggestion
   cx_tx_bytes_sent, Counter, Total payload bytes transmitted, including retransmitted bytes.
```

**Review by ggreenway — CHANGES_REQUESTED** [review]
Let me know what you think of the proposed documentation changes. I cross-referenced the comments in https://github.com/torvalds/linux/blob/master/include/uapi/linux/tcp.h with https://www.rfc-editor.org/rfc/rfc4898 to figure out exactly what those fields mean.

/wait

**Review by ggreenway — CHANGES_REQUESTED** [review]
Change looks good, but CI is failing on docs:

`ERROR: "csv-table" widths do not match the number of columns in table (4).`

I'm not sure exactly what the error is. Is there some unexpected whitespace (which github doesn't display clearly)?

/wait

**Comment by shovalyechiel1** [comment]
> Change looks good, but CI is failing on docs:
> 
> `ERROR: "csv-table" widths do not match the number of columns in table (4).`
> 
> I'm not sure exactly what the error is. Is there some unexpected whitespace (which github doesn't display clearly)?
> 
> /wait

feels like the comma in the row 'cx_tx_bytes_sent' messes it up. 
`   cx_tx_bytes_sent, Counter, Total payload bytes transmitted, including retransmitted bytes.

**Review by ggreenway — APPROVED** [review]

**Pull request merged**